### PR TITLE
ComboButton Approve Basics active when Basic move information is available

### PR DESCRIPTION
**Pull request by mkrump**
## Description

Button is active when Basic move information is complete (PPM and HHG approvals are not yet active).

## Setup

```
make client_test
make server_run
make office_client_run
```

## Code Review Verification Steps
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/n/projects/2181745/stories/163275468) 

### HHG 
Screen Shot 2019-03-12 at 9 50 42 AM

### PPM
Screen Shot 2019-03-12 at 9 47 44 AM

### CODE REVIEW

File: src/scenes/Office/MoveInfo.jsx [hunk 1]
```diff
@@ -387,20 +389,23 @@ class MoveInfo extends Component {
...
-                    }
-                  />
-                )}
+                <ToolTip
+                  disabled={ordersComplete}
+                  textStyle={'tooltiptext-large'}
+                  toolTipText={
+                    'Some information about the move is missing or contains errors. Please fix these problems before approving.'
+                  }
+                >
```
Comment (kahlouie) [inline src/scenes/Office/MoveInfo.jsx]: Love that you've separated this!

File: src/shared/ToolTip/index.test.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    const disabledToolTip = renderToolTip(toolTipProps);
+
+    it('does not render toolTipText', () => {
+      const tooltipText = disabledToolTip.find('.tooltiptext');
+
+      expect(tooltipText.exists()).toBe(false);
+    });
+  });
+
+  describe('when ToolTip is enabled', () => {
```
Comment (kahlouie) [inline src/shared/ToolTip/index.test.js]: Let's add a test that checks the text we're sending is rendered properly.

File: src/scenes/Office/MoveInfo.jsx [hunk 3]
```diff
@@ -387,20 +389,23 @@ class MoveInfo extends Component {
...
-                    buttonText={'Approve'}
-                    disabled={!ordersComplete}
-                    toolTipText={
-                      'Some information about the move is missing or contains errors. Please fix these problems before approving.'
-                    }
-                  />
-                )}
+                <ToolTip
+                  disabled={ordersComplete}
+                  textStyle={'tooltiptext-large'}
```
Comment (jim) [inline src/scenes/Office/MoveInfo.jsx]: (nit) String literals don't need to be wrapped in curly braces.

```suggestion
                  textStyle="tooltiptext-large"
```

Comment (mkrump) [reply src/scenes/Office/MoveInfo.jsx]: no worries. good to know. nit away!

File: src/shared/ToolTip/index.test.js (hunk 2 above)
Comment (mkrump) [reply src/shared/ToolTip/index.test.js]: updated.

**Pull request merged**